apps/src/StudioApp.js

```diff
@@ -360,6 +362,14 @@ StudioApp.prototype.init = function(config) {
     this.handleUsingBlockly_(config);
   }
 
+  var vizResizeBar = document.getElementById('visualizationResizeBar');
+  if (vizResizeBar) {
+    vizResizeBar.addEventListener('mousedown',
```

Comment:
these do not look touch- or ie-friendly. I think you want dom.addMouse[Up|Down]TouchEvent().